### PR TITLE
Update motorola_cedric.xml

### PR DESCRIPTION
**Pull request by luka177**
Moved to active fork.

### CODE REVIEW

File: manifests/motorola_cedric.xml [hunk 1]
```diff
@@ -1,14 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <manifest>
-	<remote name="rei"
-		fetch="https://github.com/reivilibre"
-		revision="halium-7.1" />
+	<remote name="ubp-on-cedric"
+		fetch="https://github.com/ubports-on-cedric"
```
Comment (Vince1171) [inline manifests/motorola_cedric.xml]: you mean https://github.com/halium-on-cedric ;)
https://github.com/ubports-on-cedric => 404
please amend your commit and force push

**Review by Vince1171 — CHANGES_REQUESTED** [review]

File: manifests/motorola_cedric.xml (hunk 1 above)
Comment (luka177) [reply manifests/motorola_cedric.xml]: oh lol yes, it was renamed

Comment (Vince1171) [reply manifests/motorola_cedric.xml]: do you plan to change your PR?

Comment (luka177) [reply manifests/motorola_cedric.xml]: yes i do, sorry currently my time is limited

**Comment by Vince1171** [comment]
LGTM

**Pull request merged**